src/styles/messages.scss

```diff
@@ -321,61 +247,36 @@
 		@include scrollbar;
 		overflow-x: scroll;
 	}
-}
-
-.mc-show-raw {
-	@include unselectable;
-	@include clickable;
 
-	float: right;
-	display: flex;
-	align-items: center;
-	font-size: 13px;
-	color: $primaryTextColor;
-	padding: 0 4px;
-
-	&__title {
-		font-weight: bold;
+	&__controls {
+		display: none;
+		flex-direction: column;
+		row-gap: 15px;
 	}
 
-	&__icon {
-		height: 14px;
-		width: 14px;
-		background-size: 100%;
-		margin-left: 5px;
-
-		&.expanded {
-			background-image: url(../../resources/icons/arr4-up.svg);
-		}
+	&__control-button {
+		cursor: pointer;
+		width: 15px;
+		height: 15px;
```

Comment:
You could rearrange the selectors and extract width and height to an upper level to avoid copy-pasting it.

Reply:
k, i did that with other solution: https://github.com/th2-net/th2-rpt-viewer/commit/4f08c9d2f2a05b20a8ecfcced428a53c826c6992